packages/wrangler/src/cloudchamber/index.ts

```diff
@@ -0,0 +1,39 @@
+import { type CommandModule } from "yargs";
+import { createCommand } from "./create";
+import { deleteCommand } from "./delete";
+import { RegistriesCommand } from "./images/images";
+import { listCommand } from "./list";
+import { modifyCommand } from "./modify";
+import { SSHCommand } from "./ssh/ssh";
+import type { CommonYargsOptions } from "../yargs-types";
+
+export const CloudchamberCommand: CommandModule<
+	CommonYargsOptions,
+	CommonYargsOptions
+> = {
+	command: "cloudchamber",
+	describe: "📦  Manage Cloudchamber",
+	handler: () => {},
+	builder: (yargsInit) => {
+		const yargs = deleteCommand(
+			modifyCommand(
+				createCommand(
+					listCommand(
+						yargsInit.option("json", {
+							requiresArg: false,
+							default: false,
+							type: "boolean",
+							describe: "if this is true, wrangler will output json only",
+						})
+					)
+				)
+			)
+		);
```

Comment:
This is fairly nested, and would probably benefit from being individual `.command()` calls readability-wise. To maintain the consistent `--json` flag, it might be worth following a pattern similar to `wrangler ai` for each command's options (https://github.com/gabivlj/workers-sdk/blob/f6b9c89cdf97bc1c260427640e635eeffaddc173/packages/wrangler/src/ai/options.ts#L11-L17)

Reply:
Yeah might take that method and put it on a common file.